crates/wasmi/src/store.rs

```diff
@@ -67,6 +71,28 @@ impl StoreIdx {
 /// A stored entity.
 pub type Stored<Idx> = GuardedEntity<StoreIdx, Idx>;
 
+pub struct ResourceLimiterRef<'a>(pub(crate) Option<&'a mut (dyn crate::ResourceLimiter)>);
```

Comment:
Overlooked `pub(crate)` that could be removed from what I see.

Reply:
```suggestion
pub struct ResourceLimiterRef<'a>(Option<&'a mut (dyn crate::ResourceLimiter)>);
```

Reply:
Sure. Added.